pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -27,6 +27,7 @@ import cockpit from "cockpit";
 import { superuser } from "superuser";
 import { useObject, useEvent } from "hooks.js";
 import { EmptyStatePanel } from "cockpit-components-empty-state.jsx";
+import { ListingTable } from "cockpit-components-table.jsx";
```

Comment:
This breaks the pixel text. It pulls in some table styles and the usage cards also use table but not with this styling. I'll see what I can do -hopefully just import it later? :crossed_fingers:

Reply:
Ok, so it was very tricky to fix and would lead down a rabbit hole, so I used `<List>` instead.